client/src/components/GenericContainer.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import MobileDetect from 'mobile-detect';
+import { LayerHost, Stack, Theme } from '@fluentui/react';
+import { getGenericContainerStyles } from '../styles/GenericContainer.styles';
+import { getDefaultLayerHostStyles } from '../styles/Common.styles';
+import React from 'react';
+
+interface GenericContainerProps {
+  layerHostId: string;
+  theme: Theme;
+}
+
+const GenericContainer = (props: React.PropsWithChildren<GenericContainerProps>): JSX.Element => {
```

Comment:
Here is another article about composition: https://codeburst.io/a-complete-guide-to-props-children-in-react-c315fab74e7c

Reply:
Excellent, thank you for the links.. Since this is a style component, let's either name it as such or put it in a separate folder, or move it to the existing styles folder

Reply:
It's purpose is only fr styling, yes? Currently the components under the "component" folder have served some business purpose (action) , hence the suggestion

Reply:
The purpose of the "components" folder is to house React components and this component is just one of them, regardless of their purpose. The "styles" folder contains objects used for styling components and not the React components themselves. I don't think it makes sense to put a React component in a folder that shouldn't contain React components.